cli/server.go

```diff
@@ -910,8 +910,8 @@ func (r *RootCmd) Server(newAPI func(context.Context, *coderd.Options) (*coderd.
 				return xerrors.Errorf("failed to instantiate notification manager: %w", err)
 			}
 
-			// nolint:gocritic // TODO: create own role.
-			notificationsManager.Run(dbauthz.AsSystemRestricted(ctx))
+			// nolint:gocritic // We need to run the manager in a notifier context.
```

Comment:
Does this lint error occur when not using system?

Reply:
Yep, it occurs for any of the `authz.As{Role}` functions

https://github.com/coder/coder/blob/6e54bd918387c7b591f47b82dcbd7317f39e4e95/scripts/rules.go#L23-L44

Reply:
Ha, TIL.